src/com/google/javascript/jscomp/js/es6/util/inherits.js

```diff
@@ -49,25 +49,31 @@
  *
  * @param {!Function} childCtor Child class.
  * @param {!Function} parentCtor Parent class.
+ * @param {boolean} es5Out language out is ES5
  */
-$jscomp.inherits = function(childCtor, parentCtor) {
-  /** @constructor */
-  function tempCtor() {}
-  tempCtor.prototype = parentCtor.prototype;
-  childCtor.superClass_ = parentCtor.prototype;
-  childCtor.prototype = new tempCtor();
-  /** @override */
-    childCtor.prototype.constructor = childCtor;
+$jscomp.inherits = function(childCtor, parentCtor, es5Out) {
+  if (es5Out === true && Object.setPrototypeOf) {
```

Comment:
I'm inclined to drop the `&& Object.setPrototypeOf` check in favor of this:
```javascript
if (useSetPrototypeOf) {
  if ('function' != typeof Object.setPrototypeOf) {
    throw new Error('Object.setPrototypeOf function is not defined.')
  }
```
The useSetPrototypeOf parameter is basically an assertion that Object.setPrototypeOf should be available.
If it isn't, this code should fail with an exception.